pages/admin/lessons/new/index.tsx

```diff
@@ -0,0 +1,96 @@
+import React, { useState } from 'react'
+import { gql, useLazyQuery } from '@apollo/client'
+import { AdminLessonCard } from '../../../../components/admin/lessons/AdminLessonCard'
+import { withGetApp, GetAppProps } from '../../../../graphql'
+import { AdminLayout } from '../../../../components/admin/AdminLayout'
+import Link from 'next/link'
+import styles from '../../../../scss/adminLessonPage.module.scss'
+
+const EXERCISES = gql`
+  query {
+    exercises {
+      flaggedAt
+      module {
+        lesson {
+          title
+        }
+      }
+    }
+  }
+`
+type Exercise = {
+  flaggedAt?: string
+  module: {
+    lesson: {
+      title: string
+    }
+  }
+}
+
+type Exercises = Exercise[]
+
+const Lessons: React.FC<GetAppProps> = ({ data }) => {
+  const { lessons } = data
+  const [lessonCards, setCards] = useState<[]>([])
```

Comment:
It is confusing for `lessonCards` and `setCards` to be different naming convention. Please use same naming convention for both, example `const [lessonCards, setLessonCards] = ...`